src/pages/Account/test/Account.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import 'jest-styled-components'
+
+import { App } from '../../../App'
+import urls from '../../../constants/urls'
+import { renderWithRouter } from '../../../utilsTest/render'
+import { configureStore } from '../../../store'
+import { USER } from '../../../utilsTest/mockData'
+
+describe('[pages] Account', () => {
+  describe('when not logged in', () => {
+    it('should redirect to login page', () => {
+      const renderer = renderWithRouter(<App />, urls.myAccount)
+      const HTMLDivElement = renderer.getByTestId('login-page')
```

Comment:
It might be better to test if any text content from login page is in the DOM than testing if data-testid is in DOM. Data-testid does not guarantee that page has some content, while getByText('Sign In') guarantees that user see at least part of webpage - title (user does not know what is data-testid).

I really recommend you to read article https://kentcdodds.com/blog/testing-implementation-details :muscle:

Reply:
The simmilar way can be tested other places, where data-testid occurs

Reply:
Fixed! Thanks for notice of this article. I added to start of my Chrome bookmark with articles, which should i read.